storage/cloud-client/snippets_test.py

```diff
@@ -287,3 +294,81 @@ def test_bucket_lifecycle_management(test_bucket, capsys):
     out, _ = capsys.readouterr()
     assert "[]" in out
     assert len(list(bucket.lifecycle_rules)) == 0
+
+
+def test_create_bucket_class_location():
+    bucket = storage_create_bucket_class_location.create_bucket_class_location(
+        "storage-snippets-test"
```

Comment:
Please use a UUID for bucket names to ensure they don't conflict when running multiple times